op-e2e/actions/l2_challenger_test.go

```diff
@@ -76,28 +128,48 @@ interaction:
 	require.NoError(rt.t, err)
 	require.Equal(rt.t, chal.StatusNone, status)
 
-	// check bond amount doubled after challenge proven
-	bond, err := rt.valPoolContract.GetBond(nil, rt.outputIndex)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, big.NewInt(2*rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt().Int64()), bond.Amount)
+	if version == defaultValidatorSystemVersion {
+		// check bond amount doubled after challenge proven
+		bond, err := rt.valPoolContract.GetBond(nil, rt.outputIndex)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, big.NewInt(2*rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt().Int64()), bond.Amount)
+	} else if version == defaultValidatorSystemVersion+1 {
+		// check asserter has been slashed
+		valStatus, err := rt.validator.getValidatorStatus(rt.t)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, val.StatusRegistered, valStatus)
+
+		inJail, err := rt.validator.isInJail(rt.t)
+		require.NoError(rt.t, err)
+		require.True(rt.t, inJail)
+
+		// check security council has received tax
+		bal, err := rt.assetTokenContract.BalanceOf(nil, rt.sd.DeploymentsL1.SecurityCouncilProxy)
```

Comment:
Also, why don't we check the pending challenge reward for the winner?

Reply:
We don't have getter of pending challenge reward, so we cannot check it in e2e test. Also I think getter is unnecessary for public purpose. I think it's enough to check challenge reward distribution in contract tests.

Reply:
I see. Then, how about checking if the amount of validator is decreased as expected?